src/components/Sidebar/SidebarView.js

```diff
@@ -1,12 +1,42 @@
-import React from 'react';
+import React, { Component } from 'react';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
 import Search from './Search';
 
-function SidebarView() {
-  return (
-    <div>
-      <Search />
-    </div>
-  );
+class SidebarView extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: 'search',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event, value) {
+    this.setState({ value });
+  }
+
+  render() {
+    const { value } = this.state;
+
+    return (
+      <div>
+        <Tabs
+          value={value}
+          textColor="primary"
+          indicatorColor="primary"
+          onChange={this.handleChange}
+        >
+          <Tab label="Search" />
```

Comment:
Getting console errors when I fire up the branch, I think it's because you didn't add the values to the Tab components themselves.

Reply:
Don't forget to test out any changes you make locally before you push them up, no matter how small the change!